src/NzbDrone.Core/MediaFiles/EpisodeFileMovingService.cs

```diff
@@ -68,18 +71,18 @@ public EpisodeFile MoveEpisodeFile(EpisodeFile episodeFile, Series series)
             return TransferFile(episodeFile, series, episodes, filePath, TransferMode.Move);
         }
 
-        public EpisodeFile MoveEpisodeFile(EpisodeFile episodeFile, LocalEpisode localEpisode)
+        public (EpisodeFile, bool) MoveEpisodeFile(EpisodeFile episodeFile, LocalEpisode localEpisode, ScriptImportDecisionInfo scriptImportDecisionInfo)
```

Comment:
A lot of this information is already part of `LocalEpisode`, if there is anything missing could it be added instead of passing along values for a specific import method that may not be used?

Reply:
I've been wondering about this too. Yes a lot of it is already in LocalEpisode, but basically everything added in this commit needed to come from elsewhere. I need to try the other way and see if it's alright, but I don't love having functions with 5+ parameters only used like once inside them.